src/test/java/com/wikia/webdriver/testcases/specialpagestests/UserAndRights.java

```diff
@@ -47,6 +50,28 @@ public void blockedUserShouldSeeMessageOnArticleEdit() {
     edit.verifyBlockedUserMessage();
   }
 
+  @Test(groups = {"usersAndRights002", "UsersAndRights"}, dependsOnMethods = {
```

Comment:
move group `UsersAndRights` to class groups, and the same for other tests in this class